src/model/transaction/modifyBlockForContentState.js

```diff
@@ -14,15 +14,14 @@
 'use strict';
 
 var Immutable = require('immutable');
-
 import type ContentState from 'ContentState';
-import type {DraftBlockType} from 'DraftBlockType';
 import type SelectionState from 'SelectionState';
+import type ContentBlock from 'ContentBlock';
```

Comment:
Nit: We like to alphabetize requires and imports, so `ContentBlock` should just be the first in the `import` list.